tests/io/test_bedio.py

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+from atacworks.io import bedio
```

Comment:
update the ordering of imports to be 

```
standard modules

3rd party modules

own package modules
```

e.g.
```
import os

import pandas as pd
import pytest

from atacworks.io import bedio
```

Reply:
done.